src/components/Button/Button.js

```diff
@@ -200,9 +200,9 @@ class Button extends Component<ButtonProps> {
   };
 
   onClick = (event: *) => {
-    const { onClick, disabled } = this.props;
+    const { onClick, disabled, loading } = this.props;
 
-    !disabled && onClick && onClick(event);
+    !disabled && !loading && onClick && onClick(event);
```

Comment:
@ej9x we already disable the ability to interact with `disabled` / `loading` button via CSS `pointer-events: none`. Can we delete this logic?